pkg/cloudevents/eventcontext_v1.go

```diff
@@ -48,7 +48,7 @@ type EventContextV1 struct {
 
 	// Extensions - Additional extension metadata beyond the base spec.
 	// +optional
-	Extensions map[string]string `json:"-"`
```

Comment:
this is an intended change. in 1.0, only string is supported unless I am reading the spec wrong now.